src/main/resources/META-INF/plugin.xml

```diff
@@ -10,6 +10,14 @@
       <li>Initial release</li>
     </ul>    
     Learn more in the <a href="https://github.com/redhat-developer/lsp4ij/milestone/1?closed=1">changelog</a>.
+    <h3>0.0.2</h3>
+    <ul>
+      <li>Provides menu for LSP implementation/references/declaration/type definition</li>
+      <li>Export user defined language server</li>
```

Comment:
missing lots of features from https://github.com/redhat-developer/lsp4ij/issues?q=is%3Aclosed+milestone%3A0.0.2+label%3Aenhancement

Reply:
fixed